models/comments.models.js

```diff
@@ -35,7 +39,7 @@ exports.selectCommentsByArticleId = (article_id) => {
        .then(({ rows }) => {
  
       return rows[0];
-    });
+    }).catch((error )=>{return Promise.reject({status:404 , msg :'Not Found'})});
```

Comment:
catch is used for PSQL errors here. You would normally have a catch block in your controller and do any Promise.rejects above (not in a catch)

Reply:
Also As the table constraints for comments body and author are ‘NOT NULL’ you should also get a psql error for this if no value is provided. You don't need to manually check for these